raiden_contracts/constants.py

```diff
@@ -11,6 +13,57 @@
 CONTRACT_MONITORING_SERVICE = 'MonitoringService'
 CONTRACT_RAIDEN_SERVICE_BUNDLE = 'RaidenServiceBundle'
 
+# Deployed contract information
+# Deployed to Ropsten revival on 2018-09-03 from
+# raiden-contracts@fc1c79329a165c738fc55c3505cf801cc79872e4
+ROPSTEN_REGISTRY_ADDRESS = '0xf2a175A52Bd3c815eD7500c765bA19652AB89B30'
+ROPSTEN_DISCOVERY_ADDRESS = '0xEEADDC1667B6EBc7784721B123a6F669B69Eb9bD'
+ROPSTEN_SECRET_REGISTRY_ADDRESS = '0x16a25511A92C5ebfc6C30ad98F754e4c820c6822'
+# Deployed to Ropsten revival on 2018-09-21 from
+# raiden-contracts@bfb24fed3ebda2799e4d11ad1bb5a6de116bd12d
+ROPSTEN_LIMITS_REGISTRY_ADDRESS = '0x6cC27CBF184B4177CD3c5D1a39a875aD07345eEb'
```

Comment:
`ROPSTEN_LIMITS_TOKEN_NETWORK_REGISTRY_ADDRESS` ?

Reply:
Sure, it's just a bit big :)